pkg/occlient/occlient.go

```diff
@@ -35,6 +35,7 @@ import (
 	projectclientset "github.com/openshift/client-go/project/clientset/versioned/typed/project/v1"
 	routeclientset "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
 	userclientset "github.com/openshift/client-go/user/clientset/versioned/typed/user/v1"
+	KCorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
```

Comment:
Go package names should be lowercase one word. so `kcorev1` would be better.

You shouldn't need this import at all. See my comment in`WaitForDefaultServiceAccountInNamespace` function